packages/dd-trace/src/plugins/util/web.js

```diff
@@ -461,19 +462,6 @@ function addResponseTags (context) {
   web.addStatusError(req, res.statusCode)
 }
 
-function addResourceTag (context) {
-  const { req, span } = context
-  const tags = span.context()._tags
-
-  if (tags['resource.name']) return
-
-  const resource = [req.method, tags[HTTP_ROUTE]]
-    .filter(val => val)
-    .join(' ')
-
-  span.setTag(RESOURCE_NAME, resource)
```

Comment:
Why is `RESOURCE_NAME` tag no longer added?

We're kinda-sorta using it [here](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/profiling/profilers/wall.js#L35) for establishing endpoint names to associate with profiles, although we can also compute the value from `HTTP_METHOD` and `HTTP_ROUTE` tags when `RESOURCE_NAME` isn't present.

Reply:
It's still set but later in the pipeline. At this point the resource name cannot be trusted anyway because we allow changing it up until the span is finished, which is often used by customers.